bedrock/firefox/views.py

```diff
@@ -526,3 +524,23 @@ def ios_testflight(request):
     return l10n_utils.render(request,
                              'firefox/testflight.html',
                              {'newsletter_form': newsletter_form})
+
+
+class FirefoxHubView(BlogPostsView):
+    blog_posts_limit = 1
+    blog_posts_template_variable = 'articles'
+    blog_slugs = 'firefox'
+    blog_tags = ['home']
+    template_name = 'firefox/hub/home.html'
+
+    # Is this necessary with a redirect added to firefox/redirects.py?
+    # (The redirect in firefox/redirects.py doesn't work atm, so...)
+    def dispatch(self, request, *args, **kwargs):
+        locale = l10n_utils.get_locale(request)
+
+        # until we get localizable copy, non en-US locales continue to get
+        # redirect, though now it's a 302
+        if not locale == 'en-US':
```

Comment:
NIT: This is a little confusing to read. Would be better as `if locale != 'en-US':`.